doc/rst/users-guide/base/configs.rst

```diff
@@ -144,8 +144,8 @@ they can be preserved depend on the specific shell being used and are
 beyond the scope of this article.
 
 Setting ``config`` defaults on the compiler's command-line also
-supports more radical alterations to the code.  For example, we
-could change *age* from an integral type to a floating point type:
+support more radical alterations to the code.  For example, we
+could change *age* from an integral type to a floating-point type:
```

Comment:
This change to "support" isn't correct.  The subject of the sentence is the action "Setting ...", which reads as singular, so the verb should be the singular "supports".

Reply:
Agreed. Thanks!